source/monitoring_audience/analyzing_data_usage.rst

```diff
@@ -1,90 +1,109 @@
+- Totals: figures about the datasets published in the portal and their usage through that portal
+- Popularity: statistics and charts to compute and analyze the popularity of the datasets published in the portal
+- Top 5: lists of most downloaded, used and reused published datasets
+- Theme Distribution: tree chart to analyze most and least used themes
 
-.. ifconfig:: language == 'en'
+.. admonition:: Datasets used to build the Data dashboard
+   :class: important
 
-    .. image:: usage__data-usage-1-en.jpg
-        :alt: Data Usages Popularity Indicator
+   In this dashboard, only published datasets are taken into account. Unpublished datasets are ignored from all figures, charts and other computed analytics displayed.
 
-.. ifconfig:: language == 'fr'
+   It is however possible, by clicking the buttons at the top of the dashboard, to choose to build the dashboard:
 
-    .. image:: usage__data-usage-1-fr.jpg
-        :alt: Usage Data Indicateur Popularité
+   - with all existing published datasets (checked by default)
+   - only with the private datasets, meaning those in restricted access, no matter the restriction
+   - or, only with the public datasets, meaning those accessible by everyone.
 
-2. **Total datasets**: Total number of datasets in the catalog.
+   Depending on which datasets are used to build the Data dashboard, the displayed figures, charts and analytics will vary.
 
-.. ifconfig:: language == 'en'
+Totals
+------
 
-    .. image:: usage__data-usage-2-en.jpg
-        :alt: Data Usages Total Datasets Indicator
+.. localizedimage:: data_dashboard_totals.png
+    :alt: Screenshot of the Totals section of the Data dashboard
 
-.. ifconfig:: language == 'fr'
+.. list-table::
+  :header-rows: 1
 
-    .. image:: usage__data-usage-2-fr.jpg
-        :alt: Usage Data Indicateur Total Jeux de données
+  * * Label
+    * Description
+  * * Datasets
+    * Total number of published datasets in the portal
+  * * Records
+    * Total number of records in all published datasets combined
+  * * Downloads
+    * Total number of downloads in all published datasets combined
+  * * API calls
+    * Total number of API calls for all published datasets combined
+  * * Reuses
+    * Total number of reuses for all published datasets combined
 
-3. **Total records**: Total number of records in the catalog.
+Popularity
+----------
 
-.. ifconfig:: language == 'en'
+The Popularity section of the Data dashboard is based upon a computation aiming at quantifying dataset consumption in order to establish popularity scores for datasets. This computation is based on 3 metrics: number of downloads, number of API calls and number of reuses. These 3 metrics indicate whether a dataset published into an OpenDataSoft portal can be considered popular or not, compared to the other datasets of that portal.
```

Comment:
We may want to add that the scoring formula may change in the future. 

And maybe say "compared to the other datasets of the OpenDataSoft data network" but up to you I don't want to make it more difficult to people.